client/src/components/profile/About.js

```diff
@@ -43,20 +43,22 @@ export const About = () => {
     e.preventDefault()
     if (occupation !== null && bio !== null) {
       setUpdatedDetails((updatedDetails) => {
-        return { ...updatedDetails, updatedOccupation: occupation, updatedBio: bio }
+        return {
+          ...updatedDetails,
+          updatedOccupation: occupation,
+          updatedBio: bio,
+        }
       })
-    }
-    else if (bio !== null) {
+    } else if (bio !== null) {
```

Comment:
are you sure you want to use an `else if` here? because only one statement in this block is going to get executed

Reply:
It was to limit state rerender, because originally it made multiple calls, and you commented on it. First If statement will update both if both bio and occupation had a previous value. the if else will only update either or because updatedOccupation and updatedBio are already set to empty string ''. If I query, and either field is null, then the editing field will be an empty string, if they have a value, it would be put as the value for the editing field.